src/incubator/TextField/index.tsx

```diff
@@ -99,15 +99,19 @@ const TextField = (props: InternalTextFieldProps) => {
   const hidePlaceholder = shouldHidePlaceholder(props, fieldState.isFocused);
   const retainTopMessageSpace = !floatingPlaceholder && isEmpty(trim(label));
   const centeredContainerStyle = centered && styles.centeredContainer;
-  const centeredLabelStyle = centered && styles.centeredLabel;
+  const _labelStyle = useMemo(() => (centered ? [labelStyle, styles.centeredLabel] : labelStyle),
+    [labelStyle, centered]);
+  const _validationMessageStyle = useMemo(() => {
+    return centered ? [validationMessageStyle, styles.centeredValidationMessage] : validationMessageStyle;
+  }, [validationMessageStyle, centered]);
```

Comment:
Also here, 
This change shouldn't be part of this PR.. 
please do it in a different PR next time 🙏🏽